regulament_pregatit_feedback.txt

```diff
@@ -660,6 +665,79 @@ Art. 106.
+
+Art. 107+4. În situații excepționale, în care nu este posibilă arondarea la un Centru Local din proximitatea geografică, Consiliul Director poate împuternici orice alt Centru Local eligibil să mentoreze grupul de cercetași, cu acordul acelui Centru Local. Alternativ, Consiliul Director poate împuternici nominal un grup de minim 3 lideri din ONCR care să asigure mentoratul grupului de cercetași, acesta primind formal mentoratul Consiliului Director; într-o astfel de situație excepțională, liderii și membrii grupului de cercetași vor participa la activități cercetășești și administrative în Centrele Locale ale liderilor desemnați ca mentori.
+
+Art. 107.5. Pentru înființarea unui Grup de Cercetași sunt necesare:
+	(1) Acordul unui grup informal de persoane de a forma împreună un Grup de cercetași
+	(2) Elaborarea, de către o structură mentor alături de Grupul de cercetași a unui Plan de Mentorat pe o perioadă de 1 an, care să descrie în ce condiții se va desfășura mentoratul și care conține minim:
+		(a) angajamentul reciproc al Grupului de Cercetași și al structurii mentor pentru dezvoltarea cercetășiei în Grupul de Cercetași, 
+		(b) obiective și un plan de acțiune concret cu privire la participarea Grupului de cercetași la activități cercetășești și organizatorice ale structurii mentor, precum și activitățile proprii Grupului de cercetași
+		(c) termenele pentru planificarea și evaluarea planului de mentorat
+		(d) persoanele desemnate din partea Grupului de cercetași și a stucturii mentor pentru urmărirea implementării planului de mentorat
+	(3) Acordul unei structuri de a prelua mentoratul Grupului de cercetași, în baza Planului de mentorat de la punctul (2), în condițiile art. 107+3 și 107+4; în situația din art. 107+3, acordul este dat printr-o hotărâre de Adunare Generală; în situația art. 107.4, acordul este dat printr-o decizie a Consiliului Director.
+	(4) Adițional, în cazul în care structura mentor este un Centru Local, este necesar și avizul Consiliului Director pentru Planul de mentorat.
+
+Art. 107.5. Planul de mentorat poate fi reînnoit după perioada de 1 an și în situații excepționale poate fi modificat în timpul implementării, cu acordul tuturor părților implicate, exprimat conform Art. 107.5, punctele (3) și (4).
+Art. 107.5+0. Cerințele minime pentru Planul de Mentorat sunt elemente de procedură internă a ONCR.
+
+Art. 107+5+0. Activitățile Grupului de cercetași pot avea loc doar cu participarea unor reprezentanți ai structurii mentor sau cu acordul explicit al structurii mentor dat în scris pentru fiecare activitate în parte; în această a doua situație, este obligatoriu ca liderii care organizează activitatea din Grupul de Cercetași să fi participat la activități ale Centrului Local Mentor în prealabil și să fi parcurs stagiul de inițiere în cercetășie.
```

Comment:
Aici cred că e puțin overkill să dea acordul scris pentru fiecare activitate în parte. Adică mă gândesc că după o perioadă în care fac activități împreună și apoi ”supervizați”, spre finalul anului, ar trebui să poată face activități și singurei....Nu îmi dau seama cum să o refrazez. Mă mai gândesc.

Reply:
Cred că în primul an de activitate obiectivul nostru trebuie să fie să fie cineva din Grupul Mentor cu ei. Altfel, riscam sa cadem in situatia pe care o avem acum, care este profund contraproductiva si in care ne umplem, de ani de zile, de grupuri care fac chestii dupa ureche. Chestia fara participarea reprezentantilor de la grupul mentor ar trebui sa fie o chestie exceptionala.

Reply:
De asemenea, daca grupul vrea sa faca chestii singur, ar trebui sa aiba o discutie cu mentorii sa agreeze ce si cum, care ar trebui sa fie documentata undeva. acordul scris poate fi confirmarea pe mail ca ce se propune e valid.